packages/client/components/Dashboard/LeftDashNavItem.tsx

```diff
@@ -44,13 +54,25 @@ const Label = styled('div')({
   wordBreak: 'break-word'
 })
 
+const iconLookup = {
+  arrowBack: <ArrowBack />,
+  creditScore: <CreditScore />,
+  forum: <Forum />,
+  history: <History />,
+  playlist_add_check: <PlaylistAddCheck />,
+  add: <Add />,
+  exit_to_app: <ExitToApp />,
+  group: <Group />,
+  warning: <Warning />
+}
+
 interface Props {
   className?: string
   onClick?: () => void
   label: string
   href: string
   //FIXME 6062: change to React.ComponentType
```

Comment:
Do we still need this comment?

Reply:
I'm pretty sure we don't as issue https://github.com/ParabolInc/parabol/issues/6062 was fixed by @Dschoordsch's PR https://github.com/ParabolInc/parabol/pull/7154. 

This comment appears 14 times in the app, so I'll wait for Georg to confirm that we don't need it before removing them

Reply:
Good point, thanks for bringing up! That's a cleanup we should do but I didn't back then because the PR was already quite big. I'll create an issue for it.